app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def post_likes_counter_updater
+    post.update(likes_counter: post.likes.count)
+    post.likes.count
+  end
```

Comment:
- [Optional] Just as mentioned earlier, consider using the `after_save` helper for all methods that would possibly run every time a `like` or `comment` is created. You can read more about why and how [here](https://guides.rubyonrails.org/active_record_callbacks.html).